web-ui/view/timestamps_durations.ml

```diff
@@ -94,3 +102,9 @@ let show_build ~first_step_queued_at ~total_run_time =
                (Duration.of_f total_run_time);
         ];
     ]
+
+let pp_timestamp v = Option.fold ~none:"-" ~some:to_ts_string v
+
+let pp_duration =
+  Option.fold ~none:"-" ~some:(fun v ->
+      Fmt.str "%a" Run_time.duration_pp (Duration.of_f v))
```

Comment:
Side question: what is the difference between a duration and a timestamp in this context?